setup.py

```diff
@@ -79,6 +79,7 @@ def read_requirements(filename):
     package_dir={"": "src"},
     py_modules=[os.path.splitext(os.path.basename(path))[0] for path in glob("src/*.py")],
     include_package_data=True,
+    package_data={"sagemaker": ["mp_config.yaml"]},
```

Comment:
Can we add this to [MANIFEST.in](https://github.com/aws/sagemaker-python-sdk/blob/master/MANIFEST.in) file rather?